src/main/java/picard/sam/SamAlignmentMerger.java

```diff
@@ -370,4 +417,37 @@ protected boolean isContaminant(final HitsForInsert hits) {
 
     // Accessor for testing
     public boolean getForceSort() {return this.forceSort; }
+
+    private boolean compareSequenceDictionaries(final SAMSequenceDictionary d1, final SAMSequenceDictionary d2) {
```

Comment:
I think this would be better off in `SAMSequenceDictionary` also.  There are already methods there for comparing sequence dictionaries and it would be nice to keep them all in one place.